Workade/Views&ViewModels/Map/MapInfoView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MapInfoModalView.swift
+//  Workade
+//
+//  Created by Inho Choi on 2022/11/10.
+//
+
+import NMapsMap
+import UIKit
+
+class MapInfoView: UIView {
+    init() {
+        super.init(frame: .zero)
+        self.translatesAutoresizingMaskIntoConstraints = false
+        self.backgroundColor = .white
+        self.layer.cornerRadius = 30
+        
+        setLayout()
+    }
+    
+    var titleLabel: UILabel = {
+        let title = UILabel()
+        title.translatesAutoresizingMaskIntoConstraints = false
+        title.font = .customFont(for: .captionHeadlineNew)
+        title.textColor = .black
+        return title
+    }()
+    
+    var markerImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    lazy var mapButton: UIButton = {
+        let mapButton = UIButton()
+        mapButton.translatesAutoresizingMaskIntoConstraints = false
+        mapButton.setImage(UIImage(named: "navermapbutton"), for: .normal)
+        mapButton.addAction(UIAction(title: "Go") { [weak self] _ in
+            self?.naverMapButtonTapped()
+        }, for: .touchUpInside)
+        
+        return mapButton
+    }()
+    
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setLayout() {
+        addSubview(titleLabel)
+        NSLayoutConstraint.activate([
+            titleLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 22),
+            titleLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -22),
+            titleLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 85),
+            titleLabel.trailingAnchor.constraint(equalTo: self.leadingAnchor, constant: 200 + 85)
+        ])
+        
+        addSubview(markerImage)
+        NSLayoutConstraint.activate([
+            markerImage.topAnchor.constraint(equalTo: self.topAnchor, constant: 22),
+            markerImage.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -22),
+            markerImage.trailingAnchor.constraint(equalTo: self.leadingAnchor, constant: 16 + 42),
+            markerImage.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 16)
```

Comment:
[P2] 16 + 42 방식보단 widthAnchor로 잡는 것이 다른 사람이 볼 때 빠르게 이해하기 좋을 것 같습니다. 그리고 현재 top, bottom을 22씩 설정해서 height를 비율적으로 잡고있는데, width의 경우 42라는 고정값이 있어서 추후 MarkerInfoView의 height가 디자인상으로 변경되었을 때, 추가작업이 발생하게 될 듯합니다. 
```suggestion
            markerImage.widthAnchor
                .constraint(equalTo: markerImage.heightAnchor),
            markerImage.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 16)
```
변화에 유연하도록 이렇게 height와 일치시키는 방식은 어떠신가요?

Reply:
오오 배워갑니다